test/commands/alias/list.nut.ts

```diff
@@ -7,12 +7,20 @@
 import { execCmd, TestSession } from '@salesforce/cli-plugins-testkit';
 import { expect } from '@salesforce/command/lib/test';
 
+let testSession: TestSession;
+
+function unsetAll() {
+  execCmd('sfdx alias:unset DevHub');
+  execCmd('sfdx alias:unset Admin');
+  execCmd('sfdx alias:unset user');
+}
+
 describe('alias:list NUTs', () => {
-  let testSession: TestSession;
+  testSession = TestSession.create({});
```

Comment:
nit: just make this `const testSession = TestSession.create();` and remove the `let testSession: TestSession;` above